feconf.py

```diff
@@ -348,6 +352,16 @@ def get_empty_ratings():
     },
 }
 
+# The list contains dicts with category of the classifier i.e.
+# 'StringClassifiers' or 'CodeClassifiers' and the IDs of the
+# corresponding classifiers.
+ANSWER_CLASSIFIER_CLASSES = [{
+    'category': 'StringClassifiers',
```

Comment:
We can always retroactively refactor this list to include categories if we need them and they end up being useful (simply organizing it in a meaningful way with comments may be adequate to satisfy your concerns without the extra data structure complexity).